webui/react/src/services/api-ts-sdk/api.ts

```diff
@@ -99,9 +101,10 @@ export class BaseAPI {
  * @extends {Error}
  */
 export class RequiredError extends Error {
-    name: "RequiredError"
+    override name: "RequiredError"
     constructor(public field: string, msg?: string) {
         super(msg);
+        this.name = "RequiredError"
```

Comment:
Are we suppose to change this file? Isn't this auto-generated?

Reply:
Hmm, I didn't get the issue, but maybe the previous generated code was cached?